src/Evolution/Actions/ComputeVolumeFluxes.hpp

```diff
@@ -50,6 +51,42 @@ struct ComputeVolumeFluxes {
       const ParallelComponent* const /*meta*/) noexcept {  // NOLINT const
     db::mutate_apply<typename Metavariables::system::volume_fluxes>(
         make_not_null(&box));
+    if (static_cast<bool>(
+            db::get<domain::Tags::MeshVelocity<Metavariables::volume_dim>>(
+                box))) {
+      using variables_tag = typename Metavariables::system::variables_tag;
+      using fluxes_tag =
+          db::add_tag_prefix<::Tags::Flux, variables_tag,
+                             tmpl::size_t<Metavariables::volume_dim>,
+                             Frame::Inertial>;
+      db::mutate<fluxes_tag>(
+          make_not_null(&box),
+          [](const auto fluxes_ptr, const auto& evolved_vars,
+             const tnsr::I<DataVector, Metavariables::volume_dim,
+                           Frame::Inertial>& grid_velocity) noexcept {
+            tmpl::for_each<
+                typename Metavariables::system::volume_fluxes::return_tags>(
```

Comment:
Not easily done because currently `db::add_tag_prefix` givesa `Tags::Flux<Tags::Variables<Tags::Flux<Tensors>...>>` which doesn't have a `tags_list`. I can add a comment into the code